src/D2L.CodeStyle.Analyzers/ApiUsage/DependencyInjection/DependencyRegistrationsAnalyzer.cs

```diff
@@ -208,6 +219,40 @@ ITypeSymbol type
 			return ImmutableArray.Create( registration.DependencyType );
 		}
 
+		private bool TryGetInstantiatedTypeForRegistration( DependencyRegistration registration, out ITypeSymbol instantiatedType ) {

```

Comment:
Name is a bit akward. Maybe a doc comment explaining what it does (gets the factory for `.RegisterFactory` stuff, or the concrete type when that makes sense.) I can't think of a great alternative name...